prow-configurations/config.yaml.tpl

```diff
@@ -13,29 +13,5 @@ presets:
     mountPath: /etc/service-account
     readOnly: true
 
-presubmits: # runs on PRs
-  {{ .OrganizationOrUser }}/kyma:
-  - name: prow/components/ui-api-layer
-    optional: true
```

Comment:
please rebase with master - we already have a postsubmit job

Reply:
Done